ui/src/pages/share/index.vue

```diff
@@ -1,58 +1,43 @@
 <script setup lang="ts">
 import { useI18n } from 'vue-i18n';
-import { useMessage } from 'naive-ui';
 import { nextTick, onMounted, ref } from 'vue';
 
 import Terminal from '@/components/Terminal/index.vue';
 import { useConnectionStore } from '@/store/modules/useConnection';
 import TerminalProvider from '@/components/TerminalProvider/index.vue';
 
 const { t } = useI18n();
-const message = useMessage();
 const connectionStore = useConnectionStore();
 
-const verifyValue = ref<string>('');
+const verifyValue = ref<string[]>([]);
 const showModal = ref<boolean>(false);
 
-onMounted(() => {
-  showModal.value = true;
-});
-
-const handleConfirm = () => {
-  if (!verifyValue.value) {
-    message.error(t('InputVerifyCode'));
-    return;
-  }
-
+const onFinish = () => {
   connectionStore.setConnectionState({
-    shareCode: verifyValue.value,
+    shareCode: verifyValue.value.join(''),
   });
 
   nextTick(() => {
     showModal.value = false;
   });
 };
+
+onMounted(() => {
+  showModal.value = true;
+});
 </script>
 
 <template>
   <n-modal
     v-model:show="showModal"
     preset="dialog"
     :title="t('VerifyCode')"
-    :positive-text="t('ConfirmBtn')"
     :closable="false"
+    :show-icon="false"
     :close-on-esc="false"
     :mask-closable="false"
-    @positive-click="handleConfirm"
   >
-    <n-input
-      v-model:value="verifyValue"
-      clearable
-      size="small"
-      type="password"
-      show-password-on="mousedown"
-      :placeholder="t('InputVerifyCode')"
-    />
+    <n-input-otp v-model:value="verifyValue" :length="4" size="large" class="justify-center pb-3" @finish="onFinish" />
   </n-modal>
 
   <TerminalProvider v-if="!showModal">
```

Comment:
I'm sorry, but you haven't provided any JavaScript code for review. If you could provide it here, I'd be happy to assist further.